coriolis_openstack_utils/cli/migrations.py

```diff
@@ -77,7 +77,10 @@ def take_action(self, args):
                 "batch which has equivalent endpoint details was found)")
         else:
             if args.not_drill:
-                migrations = batch_migration_action.execute_operations()
+                try:
+                    migrations = batch_migration_action.execute_operations()
+                except Exception:
+                    batch_migration_action.cleanup()
```

Comment:
For each of the following try-excepts you should:
- `LOG.warn("Error occured while XYZ. Rolling back all changes")`
- re-`raise` after cleanup